pages/blog/graphql-codegen-and-apollo-client-3.mdx

```diff
@@ -0,0 +1,176 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'Improve your Apollo-Client v3 type-safety with GraphQL Codegen',
+  author: 'dotan',
+  tags: ['graphql', 'codegen', 'typescript', 'react', 'angular', 'apollo'],
+  date: '2020-10-26',
+  description: 'Get the most out of your type-system using GraphQL Codegen',
+  image: '/blog-assets/graphql-code-generator/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+We've recently introduced improvements for GraphQL-Codegen that can quickly improve your workflow and type-safety with Apollo-Client v3.
```

Comment:
While migrating many of our clients codebases from Apollo Client 2 to Apollo Client 3, we've made a lot of improvements to GraphQL Codegen (and also fixed a lot of memory leaks in Apollo Client).

We are happy to share all those new features with everyone in the community.
We hope you would use them to quickly improve your workflow, type-safety and make your migrations easier.

And as usual, we would love to hear your feedback and ideas on how we can further improve the experience of using GraphQL and Typescript in general!